jck/compiler.lang/playlist.xml

```diff
@@ -0,0 +1,522 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TestConfig/playlist.xsd">
+	<include>../jck.mk</include>
+	<test>
+		<testCaseName>jck-compiler-lang-BINC</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>export JAVA_HOME=$(TEST_JDK_HOME)$(D)  $(AND_IF_SUCCESS) \
+	perl $(TEST_RESROOT)$(D)..$(D)..$(D)system$(D)stf$(D)stf.core$(D)scripts$(D)stf.pl  \
+	-test-root=$(Q)$(TEST_RESROOT)$(D)..$(D)..$(D)system$(D)stf;$(TEST_RESROOT)$(D)..$(D)..$(D)system$(D)openjdk-systemtest$(Q) \
+	-systemtest-prereqs=$(Q)$(JCK_ROOT)$(Q)  \
+	-java-args-setup=$(Q)$(JVM_OPTIONS)$(Q) \
+	-results-root=$(REPORTDIR)  \
+	-test=Jck -test-args=$(Q)tests=lang/BINC,jckRoot=$(JCK_ROOT),jckversion=$(JCK_VERSION),testsuite=COMPILER$(Q); \
+	$(TEST_STATUS)</command>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>jck</group>
+		</groups>
+		<subsets>
+			<subset>10+</subset>
+		</subsets>
+	</test>
+	<test>
+		<testCaseName>jck-compiler-lang-CLSS-NON-AIX</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>export JAVA_HOME=$(TEST_JDK_HOME)$(D)  $(AND_IF_SUCCESS) \
+	perl $(TEST_RESROOT)$(D)..$(D)..$(D)system$(D)stf$(D)stf.core$(D)scripts$(D)stf.pl  \
+	-test-root=$(Q)$(TEST_RESROOT)$(D)..$(D)..$(D)system$(D)stf;$(TEST_RESROOT)$(D)..$(D)..$(D)system$(D)openjdk-systemtest$(Q) \
+	-systemtest-prereqs=$(Q)$(JCK_ROOT)$(Q)  \
+	-java-args-setup=$(Q)$(JVM_OPTIONS)$(Q) \
+	-results-root=$(REPORTDIR)  \
+	-test=Jck -test-args=$(Q)tests=lang/CLSS,jckRoot=$(JCK_ROOT),jckversion=$(JCK_VERSION),testsuite=COMPILER$(Q); \
+	$(TEST_STATUS)</command>
+		<platformRequirements>^os.aix</platformRequirements>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>jck</group>
+		</groups>
+		<subsets>
+			<subset>11+</subset>
+		</subsets>
+	</test>
+	<test>
+		<testCaseName>jck-compiler-lang-CLSS-AIX</testCaseName>
```

Comment:
Do we need to separate the test into `jck-compiler-lang-CLSS-NON-AIX` and `jck-compiler-lang-CLSS-AIX`? In the original file, we only have one test `jck-compiler-lang-CLSS`. And the subset is `11` in the original file, not `11+`. Not sure which one is correct.

https://github.com/AdoptOpenJDK/openjdk-tests/blob/a11b0bb8021706b61020f97c98dc0a420b89836f/jck/runtime.lang/playlist.xml#L543

Reply:
This is a copy from the way the test was broken in jck-runtime-lang-CLSS https://github.com/AdoptOpenJDK/openjdk-tests/blob/a11b0bb8021706b61020f97c98dc0a420b89836f/jck/runtime.lang/playlist.xml#L61

Reply:
Is there a particular reason for doing the same for `jck-compiler-lang-CLSS` tests? From what I can see, `jck-compiler-lang-CLSS` did not get the chance to be executed on AIX due to build timeout. 

As this PR is about reorganizing jck targets, I would perfer not mix with other changes. This PR already contains lots of changes. If break `jck-compiler-lang-CLSS` into separate platforms is needed, we can do it another PR.

Reply:
I agree. I've collapsed the two target to only one - `jck-compiler-lang-CONV`.

Reply:
Thanks